data/remnant/remnant outfits.txt

```diff
@@ -678,10 +678,10 @@ outfit "Bellows-Class Afterburner"
 	"outfit space" -7
 	"engine capacity" -7
 	"slowing resistance" 0.01
-	"afterburner thrust" 125.0
+	"afterburner thrust" 70
```

Comment:
When I tested the nerfed version out in the merganser, it seemed to be making great leaps of acceleration to me.

Reply:
The Bellows is supposed to be useful for making "jumps" for *all* the Remnant ships, not just the Merganser. Now that we have a plethora of afterburners instead of just a couple, it would be reasonable to have a small, medium, and large versions; but at the time that was deemed to be outlandish and out of place since no-one else had small-medium-large sets.

As an aside, I would have actually called these "Jump drives" if we didn't already have something using that name. The literal only reason it is called an "Afterburner" is for clarity since it uses that key and uses those mechanics. In basically every other sense it isn't supposed to be used or treated as all the other afterburners are.